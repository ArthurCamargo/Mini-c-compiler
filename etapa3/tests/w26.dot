digraph G {
n0;
n0 -> n1;
n1;
n1 -> n2;
n1 -> n3;
n1 -> n4;
n1 -> n5;
n1 -> n6;
n2;
n2 -> n7;
n2 -> n8;
n7;
n7 [label="i"];
n8;
n8 [label="0"];
n2 [label="="];
n3;
n3 [label="i"];
n4;
n4 -> n9;
n4 -> n10;
n9;
n9 [label="i"];
n10;
n10 [label="2"];
n4 [label="="];
n5;
n5 -> n11;
n5 -> n12;
n5 -> n13;
n11;
n11 [label="i"];
n12;
n12 [label="3"];
n13;
n13 -> n14;
n13 -> n15;
n14;
n14 [label="i"];
n15;
n15 [label="4"];
n13 [label="="];
n5 [label="="];
n6;
n6 -> n16;
n6 -> n17;
n6 -> n18;
n16;
n16 [label="i"];
n17;
n17 [label="5"];
n18;
n18 -> n19;
n18 -> n20;
n19;
n19 [label="i"];
n20;
n20 [label="6"];
n18 [label="="];
n6 [label="="];
n1 [label="for"];
n0 [label="f"];

}