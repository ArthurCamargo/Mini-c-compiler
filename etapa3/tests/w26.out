0x556829200350
0x556829200350, 0x5568292008e0
0x5568292008e0
0x5568292008e0, 0x5568292004f0
0x5568292008e0, 0x556829200560
0x5568292008e0, 0x556829200670
0x5568292008e0, 0x556829200780
0x5568292008e0, 0x556829200a00
0x5568292004f0
0x5568292004f0, 0x556829200450
0x5568292004f0, 0x5568292004a0
0x556829200450
0x556829200450 [label="i"];
0x5568292004a0
0x5568292004a0 [label="0"];
0x5568292004f0 [label="="];
0x556829200560
0x556829200560 [label="i"];
0x556829200670
0x556829200670, 0x5568292005d0
0x556829200670, 0x556829200620
0x5568292005d0
0x5568292005d0 [label="i"];
0x556829200620
0x556829200620 [label="2"];
0x556829200670 [label="="];
0x556829200780
0x556829200780, 0x5568292006e0
0x556829200780, 0x556829200730
0x556829200780, 0x556829200890
0x5568292006e0
0x5568292006e0 [label="i"];
0x556829200730
0x556829200730 [label="3"];
0x556829200890
0x556829200890, 0x5568292007f0
0x556829200890, 0x556829200840
0x5568292007f0
0x5568292007f0 [label="i"];
0x556829200840
0x556829200840 [label="4"];
0x556829200890 [label="="];
0x556829200780 [label="="];
0x556829200a00
0x556829200a00, 0x556829200960
0x556829200a00, 0x5568292009b0
0x556829200a00, 0x556829200b10
0x556829200960
0x556829200960 [label="i"];
0x5568292009b0
0x5568292009b0 [label="5"];
0x556829200b10
0x556829200b10, 0x556829200a70
0x556829200b10, 0x556829200ac0
0x556829200a70
0x556829200a70 [label="i"];
0x556829200ac0
0x556829200ac0 [label="6"];
0x556829200b10 [label="="];
0x556829200a00 [label="="];
0x5568292008e0 [label="for"];
0x556829200350 [label="f"];
